src/main/resources/create_tables.sql

```diff
@@ -0,0 +1,34 @@
+create type roles as enum ('jobseeker', 'recruiter')
+
+CREATE TABLE users (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    gender VARCHAR(7) NOT NULL,
+    email VARCHAR(100),
+    status roles,
+    UNIQUE(email)
+);
+
+CREATE TABLE vacancies (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(50),
+    description TEXT,
+    salary DECIMAL(10, 2),
+);
+
+create table company (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL
+);
+
+create table response (
+    user_id BIGINT REFERENCES users(id),
+    vacancy_id BIGINT REFERENCES vacancies(id)
+);
+
+create table search (
```

Comment:
Commonly such tables where we specify a relation between tables `x` and `y` we should name them "x_y".

Reply:
I'm ready for a new review